### PR TITLE
Add PIM V2 to V3 migration doc

### PR DESCRIPTION
**Pull request by FaithOmbongi**
Continuation of PR https://github.com/microsoftgraph/microsoft-graph-docs/pull/16043

### CODE REVIEW

**Comment by FaithOmbongi** [comment]
@amanswaika, raised this branch/PR as a replacement for the PR you raised https://github.com/microsoftgraph/microsoft-graph-docs/pull/16043. This is to avoid corrupting the **main** branch in your fork.

File: api-reference/includes/pim-v2AADRoles-deprecation.md [hunk 1]
```diff
@@ -7,4 +7,4 @@ ms.localizationpriority: medium
 <!-- markdownlint-disable MD041-->
 
 >[!CAUTION]
->This version of the Privileged Identity Management (PIM) API for Azure Active Directory (Azure AD) roles is deprecated and stopped returning data on May 31, 2021. Please use the [role management](/graph/api/resources/rolemanagement?view=graph-rest-beta&preserve-view=true) API.
+>This version of the Privileged Identity Management (PIM) API for Azure Active Directory (Azure AD) roles is deprecated and stopped returning data on May 31, 2021. Use the new [role management](/graph/api/resources/privilegedidentitymanagement-root.md#migrate-from-pim-v2-to-pim-v3-apis) API.
```
Comment (amanswaika) [inline api-reference/includes/pim-v2AADRoles-deprecation.md:10]: you should not have .md in the url, it doesn't work

File: api-reference/includes/pim-v2ResourceRoles-deprecation.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+author: carolinetempleton
+ms.topic: include
+ms.localizationpriority: medium
+---
+
+<!-- markdownlint-disable MD041-->
+
+>[!CAUTION]
+>This version of the Privileged Identity Management (PIM) API for Azure resources will be deprecated soon. Please use the new [Azure REST PIM API for Azure resource roles](/graph/api/resources/privilegedidentitymanagement-root.md#migrate-from-pim-v2-to-pim-v3-apis).
```
Comment (amanswaika) [inline api-reference/includes/pim-v2ResourceRoles-deprecation.md:10]: you should not have .md in the url, it doesn't work

Comment (FaithOmbongi) [reply api-reference/includes/pim-v2ResourceRoles-deprecation.md]: Great catch! Seems the validator doesn't catch broken public links. Fixed.

File: api-reference/includes/pim-v2AADRoles-deprecation.md (hunk 1 above)
Comment (FaithOmbongi) [reply api-reference/includes/pim-v2AADRoles-deprecation.md]: Great catch! Fixed.

**Pull request merged**